zenodo/config.py

```diff
@@ -840,6 +842,10 @@ def _(x):
 
 # Search
 # ======
+#: Default ElasticSearch host
+SEARCH_ELASTIC_HOSTS = [
+    {"host": ZENODO_DOCKER_HOST}
+]
```

Comment:
Also, I think it would be helpful to explicitly configure ``SEARCH_ELASTIC_HOSTS`` here.
```
SEARCH_ELASTIC_HOSTS = [
    {"host": "localhost"}
]
```